core/chains/evm/client/pool.go

```diff
@@ -150,6 +153,13 @@ func (p *Pool) Dial(ctx context.Context) error {
 		p.wg.Add(1)
 		go p.runLoop()
 
+		if p.switchToBestNodeInterval.Seconds() > 0 && p.selectionMode != NodeSelectionMode_RoundRobin {
+			p.logger.Infof("The pool will switch to best node every %s", p.switchToBestNodeInterval.String())
+			go p.switchToBestNodeLoop()
+		} else {
+			p.logger.Debug("Best node switching is disabled")
```

Comment:
nit: why is this a different level than the other branch?

Reply:
Fixed in [`4662c81` (#10726)](https://github.com/smartcontractkit/chainlink/pull/10726/commits/4662c81ec6ea9f511c1fbbd5365875e584efd445)